[Chores #115584861] Fix minor nitpicks

elchroy opened the pull request:
A few nitpick are being fixed which include
- Add 'required' attribute for Search
- Alignment of featured restaurant details
- Positioning of the floating menu
- Drop username on user profile
- Fix hover color for links

Commit:
Add default value of -1 to restaurant_id column in add_restaurant_id_col_to_restaurants file
Modify BookingTest to include newly added bookings table columns
Update the url and see value to the newly created values
Comment test cases in RestaurantGallery that are not yet implemented

elchroy committed:
[Chores #115584861] Fix minor nitpicks

A few nitpick are being fixed which include
- Add 'required' attribute for Search
- Alignment of featured restaurant details
- Positioning of the floating menu
- Drop username on user profile
- Fix hover color for links

andela-anandaa commented on public/css/application.css:
```diff
@@ -34,23 +33,23 @@ body{
 .navbar-nav>.open>a:focus, .navbar-default .navbar-nav>.open>a:hover{
     background:rgba(0,0,0,0.5);
-    color:#BBB24D;
+    color: crimson;
```
Use color codes instead of names.

andela-anandaa commented on public/css/application.css:
```diff
@@ -59,6 +58,10 @@ body{
+.navbar {
+    position: absolute;
+    width: 100%;
+}
```
A line should be skipped after this one.

andela-anandaa commented on public/css/application.css:
```diff
@@ -69,8 +72,13 @@ body{
     color:#FFF !important;
 }
 
+.navbar-default a:hover{
```
There should be a space after the selector list i.e. `a:hover {`

andela-anandaa commented on public/css/custom.css:
```diff
@@ -380,31 +380,31 @@ NO MORE TABLES!! ;)
 
-	.hp-table table, .hp-table thead, .hp-table .hp-table tbody, .hp-table th, .hp-table td, .hp-table tr { 
-		display: block; 
+	.hp-table table, .hp-table thead, .hp-table .hp-table tbody, .hp-table th, .hp-table td, .hp-table tr {
```
Fix the indentation / alignment

andela-anandaa commented:
Roy, please fix the merge conflicts; and update me.

elchroy committed:
[Chores #115584861] Modify style colours to use hex code

The colour codes in the styling of the dashboard are modified to use hex codes rather than names of colour.
The colour names 'crimson' and 'aquamarine' are modified.

andela-anandaa commented:
Please fix the failing tests.

elchroy committed:
[Fix #115584861] Fix minor nitpicks

The username column has been removed from the users table in the database.
The tests for the username is modified to use the firstname of the user.

elchroy committed:
Merge branch 'staging' of https://github.com/andela/resly into staging

elchroy committed:
Merge branch 'staging' into chores/minor_nitpicks

elchroy committed:
Applied fixes from StyleCI

elchroy committed:
Merge pull request #163 from andela/analysis-qM1VNL

Apply fixes from StyleCI
